store/mongo/datastore_mongo.go

```diff
@@ -55,11 +51,8 @@ type MongoStoreConfig struct {
 func newClient(ctx context.Context, config MongoStoreConfig) (*mongo.Client, error) {
 
 	clientOptions := mopts.Client()
-	if config.MongoURL == nil || config.MongoURL.Scheme == "" {
-		return nil, errors.Errorf(
-			"Invalid mongoURL %s: missing scheme",
-			config.MongoURL.String(),
-		)
+	if config.MongoURL == nil {
+		return nil, errors.New("mongo: missing URL")
```

Comment:
I am actually not sure, inventory [adds the url](https://github.com/mendersoftware/inventory/blob/master/store/mongo/datastore_mongo.go#L103) and I know that it makes sense in some cases or made sense when there were places where we used only `"mongodb"` as an URL. not sure it is the case.

Reply:
Personally, I prefer a requirement for a correct URL from the configuration instead of silently modifying it. But it's a minor thing I don't really have a strong opinion about.